truffle/src/com.oracle.truffle.api/src/com/oracle/truffle/api/TruffleLanguage.java

```diff
@@ -1036,6 +1036,14 @@ protected Object getLanguageGlobal(C context) {
         return AccessAPI.engineAccess().createDefaultLexicalScope(node, frame);
     }
 
+    protected List<String> getCompletionTriggerCharacters() {
+        return Collections.emptyList();
+    }
+
```

Comment:
We need `getSignatureTriggerCharacters()` to resolve TODO in `LanguageServerImpl`.